pkg/cmd/server/api/validation/validation.go

```diff
@@ -93,6 +93,21 @@ func ValidateRemoteConnectionInfo(remoteConnectionInfo api.RemoteConnectionInfo)
 	return allErrs
 }
 
+func ValidatePodManifestConfig(podManifestConfig *api.PodManifestConfig) fielderrors.ValidationErrorList {
+	allErrs := fielderrors.ValidationErrorList{}
+
+	if len(podManifestConfig.Path) > 0 {
```

Comment:
If the podmanifestconfig is present, the path and check interval should be required